lib/main.dart

```diff
@@ -1,67 +1,305 @@
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const HomePage(),
     );
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({Key? key, required this.title}) : super(key: key);
-  final String title;
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key}) : super(key: key);
 
   @override
-  State<MyHomePage> createState() => _MyHomePageState();
+  _HomePageState createState() => _HomePageState();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+Widget _imageContainerStart(BuildContext context, String link, String gameName,
+    String gameShortDescription) {
+  var phoneSize = MediaQuery.of(context).size;
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.9;
+    orientationHeight = 1.24;
+    marginDivider = 2;
+  }
+  return Container(
+      width: phoneSize.width * 0.83 * orientationWidth,
+      height: phoneSize.height * 0.45 * orientationHeight,
+      margin: const EdgeInsets.all(16.0),
+      decoration: BoxDecoration(
+          borderRadius: const BorderRadius.all(Radius.circular(5.5)),
+          image: DecorationImage(image: AssetImage(link), fit: BoxFit.cover)),
+      child: Container(
+        margin: EdgeInsets.only(
+            left: 12.0, right: 17.0, top: 140.0 / marginDivider),
+        child: Column(
+          children: [
+            Align(
+                alignment: Alignment.centerLeft,
+                child: Text(
+                  gameName,
+                  style:
+                      const TextStyle(fontSize: 42, color: Color(0xFFf2f2f2)),
+                )),
+            Align(
+                alignment: Alignment.centerLeft,
+                child: Text(
+                  gameShortDescription,
+                  style:
+                      const TextStyle(fontSize: 20, color: Color(0xFFf2f2f2)),
+                )),
+          ],
+        ),
+      ));
+}
 
-  void _incrementCounter() {
-    setState(() {
-      _counter++;
-    });
+Widget _imageContainerSecond(BuildContext context, String link, String gameName,
+    String gameShortDescription) {
+  var phoneSize = MediaQuery.of(context).size;
+  if (gameName.length > 9) {
+    gameName = gameName.substring(0, 9) + "...";
+  }
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.8;
+  }
+  return Container(
+    margin:
+        const EdgeInsets.only(left: 10.0, right: 10.0, top: 10.0, bottom: 20.0),
+    child: Column(children: [
+      Container(
+        width: phoneSize.width * orientationWidth * 0.83 * 1.2 / 5,
+        height: phoneSize.width * orientationWidth * 0.83 * 1.2 / 5,
+        margin: const EdgeInsets.only(
+            left: 28.0, right: 28.0, top: 2.0, bottom: 10.0),
+        decoration: BoxDecoration(
+            border: Border.all(
+              color: Colors.indigo, //kHintColor, so this should be changed?
```

Comment:
If that does not throw compile-time errors, then it's probably fine, consider removing the comment.